### ISSUE TITLE
ui/run-dev-server Uncaught TypeError from protobufjs util.LongBits 

### ISSUE THREAD

**Issue by GeoffZhu** [google/perfetto#360]
Hi，I just follow the Perfetto build instructions to build the UI，but got an Uncaught TypeError.

``` bash
git clone https://android.googlesource.com/platform/external/perfetto/
tools/install-build-deps --ui
ui/run-dev-server
```
After run these commands，and then i see `First build completed! Starting HTTP server on http://127.0.0.1:10000`

But when i open it, i got an error,  seems came from protobufjs pkg

```
Uncaught TypeError: Cannot read properties of undefined (reading 'LongBits')
    at writer.js:8:22
    at index.ts:359:8
```

Does anyone have the same problem? 

<img width="1576" alt="image" src="https://user-images.githubusercontent.com/10768286/194744157-18772cc4-3787-4174-a6bd-b95d276d1710.png">

Env
``` 
MacOS 10.15.7
```

**Comment by LalitMaganti** [comment]
@chromy have you seen this before?

**Comment by GeoffZhu** [comment]
Fixed by update @rollup/plugin-commonjs 14.0.0 to 21.0.1

Step

edit ui/package.json  dependencies @rollup/plugin-commonjs to 21.0.1

``` bash
cd ui 
npm i
npm i --package-lock-only
cd ..
```

then 

``` bash
tools/install-build-deps --ui
ui/run-dev-server
```

**Comment by primiano** [comment]
that is really weird. install-build-deps invokes `npm ci` which should stick precisely to the pinned version.
Which doesn't explain how come this doesn't happen with all the other environments where we build perfetto.
`¯\_(ツ)_/¯`
